pkg/services/grpcserver/interceptors/auth.go

```diff
@@ -101,6 +115,14 @@ func (a *Authenticator) getSignedInUser(ctx context.Context, token string) (*use
 		return nil, fmt.Errorf("service account is disabled")
 	}
 
+	if signedInUser.Permissions[signedInUser.OrgID] == nil {
```

Comment:
```suggestion
	if signedInUser.Permissions == nil || signedInUser.Permissions[signedInUser.OrgID] == nil {
```

Reply:
added as part of [ebd85d4](https://github.com/grafana/grafana/pull/56059/commits/ebd85d41fbaa044ba865a27b230e2f3948948ca6)